### PR TITLE
doc for saving and loading

### PR DESCRIPTION
**Pull request by sid597**

### CODE REVIEW

File: docs/developer_guide/self-hosted.md [hunk 1]
```diff
@@ -168,4 +168,76 @@ Once your droplet has been created, open the console to ssh to the server, follo
 
 ![digital-ocean-console-and-ip-address](https://user-images.githubusercontent.com/8952138/141150925-9f8df004-faa0-4fbe-9875-c276d60c5118.jpg)
 
+## Backup your server 
+
+NOTE : Currently this is very hands-on implementation for the backup, please feel free to ask questions in our Discord server.
+
+### Prequisetes
+  To use backup one needs to have:
+   - Athens github repo
```
Comment (neotyk) [inline docs/developer_guide/self-hosted.md:177]: `git clone`d Athens repository?

File: docs/developer_guide/self-hosted.md [hunk 2]
```diff
@@ -168,4 +168,76 @@ Once your droplet has been created, open the console to ssh to the server, follo
...
+ - goto `/var/lib/athens/backups` and find the name of file you want to get
+     - `cd /var/lib/athens/backups`
+     - `ls` : to see all the files and copy the filename
+ - log out of the serve: `ctrl-d`
+ - Get the server log to local machine
+     - NOTE : 
+       - Replace the "$REMOTE_IP" with the actual ip also 
+       - Replace the "$FILENAME" with the actual filename
+       - Replace the "$LOCAL_DIR" with the path of dir where you want to copy the save (Backup) file in your local machine.
+     - `rsync -avzh root@"$REMOTE_IP"46.101.61.240:/var/lib/athens/backups/"$FILENAME" "$LOCAL_DIR"`
```
Comment (neotyk) [inline docs/developer_guide/self-hosted.md:238]: Actually `scp` is easier, and we already assume that user has `ssh`.

**Review by neotyk — APPROVED** [review]

**Review by jefftangx — APPROVED** [review]
Good job @sid597. I will update instructions to reformat and switch `rsync` with `scp`

File: docs/developer_guide/self-hosted.md (hunk 1 above)
Comment (sid597) [reply docs/developer_guide/self-hosted.md:177]: yes I mean cloned athens repo, thank you.

File: docs/developer_guide/self-hosted.md (hunk 2 above)
Comment (sid597) [reply docs/developer_guide/self-hosted.md:238]: yeah you are right

**Pull request merged**